spec/lib/solidus_subscriptions/processor_spec.rb

```diff
@@ -108,6 +108,17 @@
       end
     end
 
+    context "when the config 'clear_past_installments' is enabled" do
+      it 'clears the past failed installments' do
+        SolidusSubscriptions.configuration.clear_past_installments = false
```

Comment:
Can you use `allow` instead of setting the configuration value directly? The reason is that setting it directly will persist it across specs, and it may "leak" to other tests, breaking them. `allow` calls, on the other hand, are re-set at the end of each test.

Reply:
Switched to `allow` for better safety